src/cards/summon_generic.ts

```diff
@@ -27,181 +27,195 @@ import { DARK_PRIEST_ID } from '../entity/units/darkPriest';
 
 
 const overrides: { [unitId: string]: { exclude: boolean, properties: { manaCost?: number } } } = {
-    'decoy': {
-        exclude: true,
-        properties: {}
-    },
-    [golem_unit_id]: {
-        exclude: false,
-        properties: { manaCost: 60 }
-    },
-    [ARCHER_ID]: {
-        exclude: false,
-        properties: { manaCost: 60 }
-    },
-    [ANCIENT_UNIT_ID]: {
-        exclude: false,
-        properties: { manaCost: 50 }
-    },
-    [GLOP_UNIT_ID]: {
-        exclude: false,
-        properties: { manaCost: 80 }
-    },
-    [gripthulu_id]: {
-        exclude: false,
-        properties: { manaCost: 80 }
-    },
-    [BLOOD_GOLEM_ID]: {
-        exclude: false,
-        properties: { manaCost: 100 }
-    },
-    [POISONER_ID]: {
-        exclude: false,
-        properties: { manaCost: 100 }
-    },
-    [VAMPIRE_ID]: {
-        exclude: false,
-        properties: { manaCost: 150 }
-    },
-    [BLOOD_ARCHER_ID]: {
-        exclude: false,
-        properties: { manaCost: 130 }
-    },
-    [PRIEST_ID]: {
-        exclude: false,
-        properties: { manaCost: 130 }
-    },
-    [SUMMONER_ID]: {
-        exclude: false,
-        properties: { manaCost: 150 }
-    },
-    [GHOST_ARCHER_ID]: {
-        exclude: false,
-        properties: { manaCost: 180 }
-    },
-    [MANA_VAMPIRE_ID]: {
-        exclude: false,
-        properties: { manaCost: 180 }
-    },
-    [DARK_SUMMONER_ID]: {
-        exclude: false,
-        properties: { manaCost: 210 }
-    },
-    [DARK_PRIEST_ID]: {
-        exclude: false,
-        properties: { manaCost: 210 }
-    },
-    [bossmasonUnitId]: {
-        exclude: false,
-        properties: { manaCost: 400 }
-    },
-    'Spellmason': {
-        exclude: false,
-        properties: {}
-    },
+  'decoy': {
+    exclude: true,
+    properties: {}
+  },
+  [golem_unit_id]: {
+    exclude: false,
+    properties: { manaCost: 60 }
+  },
+  [ARCHER_ID]: {
+    exclude: false,
+    properties: { manaCost: 60 }
+  },
+  [ANCIENT_UNIT_ID]: {
+    exclude: false,
+    properties: { manaCost: 50 }
+  },
+  [GLOP_UNIT_ID]: {
+    exclude: false,
+    properties: { manaCost: 80 }
+  },
+  [gripthulu_id]: {
+    exclude: false,
+    properties: { manaCost: 80 }
+  },
+  [BLOOD_GOLEM_ID]: {
+    exclude: false,
+    properties: { manaCost: 100 }
+  },
+  [POISONER_ID]: {
+    exclude: false,
+    properties: { manaCost: 100 }
+  },
+  [VAMPIRE_ID]: {
+    exclude: false,
+    properties: { manaCost: 150 }
+  },
+  [BLOOD_ARCHER_ID]: {
+    exclude: false,
+    properties: { manaCost: 130 }
+  },
+  [PRIEST_ID]: {
+    exclude: false,
+    properties: { manaCost: 130 }
+  },
+  [SUMMONER_ID]: {
+    exclude: false,
+    properties: { manaCost: 150 }
+  },
+  [GHOST_ARCHER_ID]: {
+    exclude: false,
```

Comment:
Didn't you mention that you added green glop here? Its still missing

Reply:
@SoulMuncher

Reply:
Sorry for the conufsion, I did not add it here, I only changed the miniboss scaling to be the same for overwritten units and those unaccounted for